CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -16,6 +16,12 @@ public class ChromiumWebBrowser : Control, IWebBrowserInternal, IWinFormsWebBrow
         private ManagedCefBrowserAdapter managedCefBrowserAdapter;
         private ParentFormMessageInterceptor parentFormMessageInterceptor;
 
+        /// <summary>
+        /// Set to true while handing an activating WM_ACTIVATE message.
+        /// MUST ONLY be cleared by DefaultFocusHandler.
+        /// </summary>
+        internal bool IsActivating = false;
```

Comment:
I suppose you're right and it should be available to be set by their own `IFocusHandler`. I'll fix it up.